### ISSUE TITLE
Support opacity for background tabs

### ISSUE THREAD

**Issue by controversial** [henrikdahl/hyper-tabs-enhanced#7]
I'm using a `hyper` config where my window is transparent and vibrant:

<img width="1015" alt="screen shot 2017-02-20 at 12 29 04 pm" src="https://cloud.githubusercontent.com/assets/10377391/23136073/06724d1e-f769-11e6-9deb-d1fad12b8ac8.png">

`hyper-tabs-enhanced` sets the background tab to opaque black, instead of a darker but still transparent look that I would expect. Perhaps for transparent black tabs, it should just increase background tab opacity?

**Comment by henrikruscon** [comment]
Can I get your config so I can try your setup? ✌️

**Comment by controversial** [comment]
[sure](https://gist.githubusercontent.com/controversial/d08d16c4fd1c9df81e722db3913c6dcf/raw/336abeea55f5cc997bf8138c8af3d1cd1d0fcacf/.hyper.js) :)

**Comment by henrikruscon** [comment]
<img width="690" alt="skarmavbild 2017-02-20 kl 18 48 00" src="https://cloud.githubusercontent.com/assets/1430576/23136468/24162f6e-f79d-11e6-84e0-a61879d799c4.png">

<img width="596" alt="skarmavbild 2017-02-20 kl 18 49 34" src="https://cloud.githubusercontent.com/assets/1430576/23136511/5a7abb42-f79d-11e6-8c90-3bb06f70e943.png">

It is transparent.

**Comment by controversial** [comment]
Ah, so it is. I think it's darker than I'd like, though, could there be a config option for "degree of darkening?" Maybe I'll work on a PR?

**Comment by henrikruscon** [comment]
If there's a way to detect vibrancy and then based on that darken the color less, that would be ideal.

**Comment by controversial** [comment]
https://github.com/matheuss/hyper-simple-vibrancy/blob/master/index.js#L1 sets the vibrancy inside the `onwindow` method, I'm not sure whether it's possible to access `browserWindow` outside of this.

**Comment by henrikruscon** [comment]
It is but I can't see that it holds the vibrancy variable. I'm gonna look into it more

**Comment by controversial** [comment]
If there's a `setVibrancy` method I'd assume there was a corresponding `getVibrancy` method

**Comment by henrikruscon** [comment]
Yeah, sadly there isn't. https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winsetvibrancytype-macos

**Comment by controversial** [comment]
Strange. Maybe a config option is the best way to go, then

**Comment by henrikruscon** [comment]
I've got a lot of work so I won't be able to look into this for like 3 days ✌️

**Comment by controversial** [comment]
That's fine, there's no rush

**Comment by outaTiME** [comment]
Same when using `hyper-transparent-dynamic`

**Comment by jbergstroem** [comment]
Can confirm I get this as well. As much as I enjoy the unique features of this plugin I can't help but feel its too opinionated. It changes colors of my background, my inactive tabs and also interferes with icons. I guess its a combo plugin that sits well with the author and that's cool too.
